tests/exporters/test_onnx.py

```diff
@@ -48,6 +62,9 @@ def test_export_onnx_sklearn_head():
         # Compare the results and ensure that we get the same predictions.
         assert np.array_equal(onnx_preds, pytorch_preds)
 
+    except Exception as e:
+        raise e
+
```

Comment:
```suggestion
```
I think this is equivalent to not writing this. If there's an error in this try-finally, it'll get raised even if this `except` is not here.